aim/cli/watcher_cli.py

```diff
@@ -0,0 +1,171 @@
+import click
+from click import core
+import uuid
+import json
+
+from aim.ext.notifier import get_config
+from aim.ext.notifier.utils import has_watcher_config, set_default_config
+from aim.sdk.run_status_watcher import RunStatusWatcher
+from aim.sdk.repo import Repo
+
+core._verify_python3_env = lambda: None
+DEFAULT_MESSAGE_TEMPLATE = "❗️ Something wrong with Run '{run_hash}'. Please check. ❗️"
+
+
+def check_configuration(repo: Repo):
+    if not has_watcher_config(repo.path):
+        if click.confirm(f'Repo {repo.path} has no configured notifications. Would you like to '
```

Comment:
"Would you like to use default configuration?" I just tapped "n" cause wanted to configure notifications. A msg like "would you like to configure .. ?" would be more helpful i guess.